pytket_qir/utils/utils.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class WASMType(Enum):
+    INT32 = 32
+    INT64 = 64
```

Comment:
This feels like a hack, attaching semantic significance to the enum values. (E.g. what if we want to add other 32-bit WASM types in the future?) Better not to rely on this in the code..

Reply:
OK, I can try to think of another solution here.

Reply:
Done.